packages/migrate/src/commands/DbExecute.ts

```diff
@@ -104,8 +99,8 @@ ${chalk.bold('Examples')}
       return this.help()
     }
 
-    if (!args['--preview-feature']) {
-      throw new DbExecuteNeedsPreviewFeatureFlagError()
+    if (args['--preview-feature']) {
+      console.warn('--preview-feature is deprecated and will be removed in the next major version.')
```

Comment:
```suggestion
      logger.warn(`"prisma db execute" was in Preview and is now Generally Available.
You can now remove the ${chalk.red('--preview-feature')} flag.`)
```
Something like this maybe, also use our `logger.warn()`, useful for Redwood for example so they can disable the warnings.